.github/ISSUE_TEMPLATE/20-feature-request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature Request
+about: Use this template for raising a feature request
+labels: 'type:feature'
+
+---
+
+<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>
```

Comment:
do we have a separate policy, or we always refer to tensorflow policy?

Reply:
@pyu10055 I am not sure , but for example [here](https://groups.google.com/a/tensorflow.org/g/tfjs)  we refer to Tensorflow code of conduct

Reply:
I can also copy the same [file](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md) in to our repository ? please let me know ?